site/content/en/docs/Reference/gameserver.md

```diff
@@ -183,27 +108,6 @@ The `spec` field is the actual GameServer specification and it is composed as fo
   - `grpcPort` the port that the SDK Server binds to for gRPC connections
   - `httpPort` the port that the SDK Server binds to for HTTP gRPC gateway connections
 - `template` the [pod spec template](https://v1-13.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#podtemplatespec-v1-core) to run your GameServer containers, [see](https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/#pod-templates) for more information.
```

Comment:
Should be more recent version:
https://v1-14.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/